### ISSUE TITLE
Owncast admin barely usable on mobile screens

### ISSUE THREAD

**Issue by rmens** [owncast/owncast#3111]
### Share your bug report, feature request, or comment.

Wanted to check on how a popular livestream was doing while traveling. The admin was barely usable on mobile with a lot of elements overlapping each other.

A lay-out with a sidebar doesn't have to be unusable on mobile. I recommend checking out how Wordpress does this.

This is how it looks on an iPhone 14 Pro:

![owncast mob](https://github.com/owncast/owncast/assets/6742496/7c312755-490e-47f3-959a-73ff008349df)

**Comment by gabek** [comment]
Yeah, the admin has never had any work to support mobile. I think it's a rare use case, but it's a valid one. At some point in the future this should be improved.
